compiler/src/dotty/tools/dotc/ast/Desugar.scala

```diff
@@ -1130,6 +1216,10 @@ object desugar {
         "given_" ++ inventTypeName(impl)
     str.toTermName.asSimpleName
 
+  /** Extract a synthesized given name from a type tree. This is used for
+   *  both anonymous givens and (under x.modularity) deferred givens.
+   *  @param followArgs   If true include argument types in the name
```

Comment:
```suggestion
   *  @param followArgs  If true include argument types in the name.
```

Reply:
Actually, Javadoc conventions disagree. And it should be lower case `if`.